### PR TITLE
feat: retry

### PR DESCRIPTION
**Pull request by NotEvenANeko**
#13

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,7 +19,6 @@
     "next": "13.4.7",
     "react": "18.2.0",
     "react-dom": "18.2.0",
-    "react-icons": "^4.10.1",
```
Comment (TinsFox) [inline package.json:22]: This package does not seem to be used 🤔

Comment (NotEvenANeko) [reply package.json:22]: It was added in #8, but the `GrPowerCycle` from it has `stroke="#000"` in its `path` so I copied its code and removed the package.

Comment (TinsFox) [reply package.json:22]: I was wrong, no problem.

**Review by ly1ex — APPROVED** [review]
LGTM

**Pull request merged**